include/cbl/CBLReplicator.h

```diff
@@ -141,6 +141,11 @@ typedef struct {
     CBLEndpoint* endpoint;              ///< The address of the other database to replicate with
     CBLReplicatorType replicatorType;   ///< Push, pull or both
     bool continuous;                    ///< Continuous replication?
+    //-- Retry Logic:
+    int maxRetries;                     ///< Max retry attempts (-1 for default, 9 for single-shot and max for continuous; 0 for no retries)
```

Comment:
I don't understand what "9 for single-shot and max for continuous" means...?

Reply:
Agreed that using -1 for default is a bit confusing. As the `maxRetries` (also `maxRetriesWaitTime`) has been implemented in all platforms. We will need to go back to the API review process again to change this. I have created the ticket to revise that. For this PR, please accept this until we decide to change the name.

Ticket : https://issues.couchbase.com/browse/CBL-1906